tinymce-language-selector/plugin.js

```diff
@@ -140,6 +140,25 @@ tinymce.PluginManager.add('language', function(editor) {
     });
   };
 
+  const addIdAttribute = (editor, attempts = 0) => {
+    // Only attempt this at most 3 times including the first pass (~ 3 seconds total)
```

Comment:
super nit: might be 2 seconds? We'll fire at 0, 1, and 2 seconds. Though there are delays in there while the event loop processes other stuff. So who knows.